src/script.js

```diff
@@ -1,3 +1,47 @@
 "use strict";
 
 // TODO - write your code here.
+
+const randomDamage = () => Math.floor(Math.random() * 10) + 1;
+
+const chooseOption = (opt1, opt2) => {
+    const randNum = Math.round(Math.random());
+    return randNum === 0 ? opt1 : opt2;
+}
+
+const attackPlayer = (health) => health - randomDamage();
+
+const logHealth = (player, health) => console.log(`${player} health: ${health}`);
+
+const logDeath = (winner, loser) => console.log(`${winner} defeated ${loser}`);
+
+const isDead = (health) => health <= 0;
+
+const fight = (
```

Comment:
nice use of arrow syntax in your function expressions. check for understanding: how come sometimes the keyword "return" is required, and other times, it is not?